rdm/init_files/documents/software_plan.md

```diff
@@ -414,7 +438,7 @@ When a new version of the software is released, the git commit corresponding to
 
 Archived releases shall be kept until there are no longer supported devices being used that run the version of the software.
 
-[[:This section fulfills 62304:5.8.7; note that documentation and configuration items are archived automatically due to the fact that they are stored in Git]]
+[[:This section fulfills 62304:5.8.7.a and 62304:5.8.7.b ; note that documentation and configuration items are archived automatically due to the fact that they are stored in Git]]
```

Comment:
it seems like there is an extra space between the "b" and the ";". Is this required due to how `rdm gap` works?

Reply:
In my mind, yes. In the code, no. Fixed.